public/js/util/networkRequest.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+function networkRequest(url) {
+  return fetch(`/get?url=${url}`)
```

Comment:
This seems closely the same as `httpGet` in the server, I think there is an isomorphic-fetch package or something that works in browser and server and we could potentially reuse this function.